DigitalOcean.API/Models/Requests/ForwardingRule.cs

```diff
@@ -1,7 +1,7 @@
 ﻿using Newtonsoft.Json;
 
-namespace DigitalOcean.API.Models.Responses {
-	public class ForwardingRules {
+namespace DigitalOcean.API.Models.Requests {
+	public class ForwardingRule {
```

Comment:
Renamed class to ForwardingRule (not plural) since it operates on a single item.